WooCommerce/Classes/Extensions/String+Woo.swift

```diff
@@ -22,6 +22,10 @@ extension String {
 /// String: URL manipulation
 ///
 extension String {
+    var hasValidSchemeForBrowser: Bool {
+        hasPrefix("http://") || hasPrefix("https://")
+    }
+
     func addHTTPSSchemeIfNecessary() -> String {
         if self.hasPrefix("http://") || self.hasPrefix("https://") {
```

Comment:
nit: maybe this can also call the new function?
```suggestion
        if hasValidSchemeForBrowser {
```